en/editor/project/index.md

```diff
@@ -86,6 +86,10 @@ Used to configure various parameters of physics, please refer to the [Physics Co
 
 - **MAX_LABEL_CANVAS_POOL_SIZE**: set the maximum number of Canvas object pools used by the Label, and adjust it according to the number of Labels in the same scene of the project.
 
+- **ENABLE_WEBGL_HIGHP_STRUCT_VALUES**：whether to enable WebGL highp precision with fragment shader struct variables. These variables are mediump precision with Android+WebGL platform. It may lead to light calculation errors of some data requiring highp precision (such as position)。
```

Comment:
```suggestion
- **ENABLE_WEBGL_HIGHP_STRUCT_VALUES**: on the Android platform with WebGL backend, the internal variables of the structures defined in the fragment shader use **mediump** precision, which may lead to incorrect results for some data that requires high precision (such as position information). To avoid this problem, check this option (new in v3.4.1) and enable WebGL to calculate variables with **highp** precision.

```